### ISSUE TITLE
Auto upload from selected folders

### ISSUE THREAD

**Issue by hardwareadictos** [Piwigo/piwigo-flutter-app#100]
Could be nice to auto upload Photo files from selected folders, for example DCIM for backups

**Comment by remi-martin** [comment]
Hi, thanks for your feedback !

This is a know feature we would like to implement on Piwigo NG, but it is not what we are looking for right now. iOS app implemented this feature after years of existence and Piwigo NG is not at that level.

**Comment by xpseudonym** [comment]
Thanks for listening :smile:

**Comment by tanty** [comment]
I think this is a duplicate of #12 ?

**Comment by xpseudonym** [comment]
Yup, looks like it to me - but, issue #100 sounds much more important than issue #12 - so, I vote to close #12 for #100... :laughing:

**Comment by xpseudonym** [comment]
Ditto issue #52
